sgt-back-end/index.js

```diff
@@ -0,0 +1,159 @@
+const express = require('express');
+const app = express();
+const pg = require('pg');
+
+app.use(express.json());
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades', (req, res) => {
+  const params = [];
+  const sql = `
+  select *
+    from "grades"
+    `;
+  db.query(sql, params)
+    .then(result => {
+      res.json(result.rows);
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        error: 'An unexpected error occured.'
+      });
+    });
+});
+
+app.post('/api/grades', (req, res) => {
+  const student = req.body;
+  student.score = parseInt(student.score);
+  if (student === 'undefined' || !Number.isInteger(student.score) ||
+    student.score > 100 || student.score < 0 ||
+    (student.name || student.course || student.score) === 'undefined') {
+    res.status(400).json({
+      error: 'Content must include valid course, name, and score'
+    });
+    return;
+  }
```

Comment:
`student` will never equal the string `'undefined'` you would need to use `typeof` for that. It would be better to just use `!student`. This should also be broken up into 2 `if` statements so you can send back more specific errors. You should have an `if` that ensures all of the properties are there, then a second `if` statement the checks if `score` is valid and in range of 0 - 100.